app/scss/addQuestion.scss

```diff
@@ -0,0 +1,91 @@
+@import 'imports/variables';
+@import url('https://fonts.googleapis.com/css?family=Asap&display=swap');
+
+* {
+  box-sizing: border-box;
+
+  body {
+    font-family: 'Asap', sans-serif;
```

Comment:
This is not applying the asap font

Reply:
I've removed the import and call. The font will be implemented properly once the story2 branch is up to date.